token/cli/src/main.rs

```diff
@@ -54,7 +53,7 @@ use std::{
 use strum_macros::{EnumString, IntoStaticStr, ToString};
 
 mod config;
-use config::{Config, KeypairOrPath, MintInfo};
+use config::{Config, MintInfo};
```

Comment:
Since these creators were moved, you need to get rid of the imports for `ProgramRpcClient*` in this file as mentioned by clippy just above

Reply:
Whoops sorry it's that they should be in the `tests` module for the test config